### ISSUE TITLE
Any way to install forks named other than "openpilot"?

### ISSUE THREAD

**Issue by LPFchan** [sshane/openpilot-installer-generator#7]
I'm currently setting up OnePlus 3T + White Panda which requires older version of the openpilot. But for the branches I wish to use the devs have decided to split out into separate repo. 

Such as:
https://github.com/crwusiz/openpilot_v0.8
https://github.com/openpilotusers/openpilot_081

Is there any way to install forks named other than "openpilot"?

**Comment by dhimmel** [comment]
Also curious how I'd use https://smiskol.com/fork/ to install the first repo listed and not the second:

- https://github.com/sunnyhaibin/openpilot-1
- https://github.com/sunnyhaibin/sunnypilot

**Comment by sshane** [comment]
I suppose we could support repo by keyword, for ex: fork/sunnyhaibin/r/openpilot-1/master

Someone would need to implement this

But then no branches can be named r, so some thought into the syntax needs to be done

**Comment by northportio** [comment]
What's the status on this as this is something I'd very much like to see

**Comment by mouxangithub** [comment]
You can try running the PHP scripts for the project (under the fork) on your own server, then modify build_agnos.php to replace "https://github.com/27182818284590452353602874713526624977572470936999595" with another repository address. The operational significance of this replacement is to substitute the repository URL, for example replacing it with "https://github.com/commaai/openpilot.git". You can replace it with any other repository name

**Comment by mouxangithub** [comment]
> I suppose we could support repo by keyword, for ex: fork/sunnyhaibin/r/openpilot-1/master
> 
> Someone would need to implement this
> 
> But then no branches can be named r, so some thought into the syntax needs to be done

Yes, I've considered that too. If using repository naming, the required input becomes longer and more cumbersome. But strangely, when I replaced the repository address with China's gitee.com, it sometimes worked but other times just skipped directly. Since I'm a Chinese player, GitHub isn't very accessible for us in China. Some car enthusiasts without VPN access might be unable to download op
